packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1521,6 +1566,61 @@ export class TransactionController extends BaseController<
   ): boolean {
     return remoteGasUsed !== localGasUsed;
   }
+
+  private requestApproval(txMeta: TransactionMeta) {
+    const id = this.getApprovalId(txMeta);
+    const { origin } = txMeta;
+    const type = ApprovalType.Transaction;
+
+    this.messagingSystem
+      .call(
+        'ApprovalController:addRequest',
+        {
+          id,
+          origin: origin || ORIGIN_METAMASK,
+          type,
+          requestData: {
+            id: txMeta.id,
+            origin: origin || ORIGIN_METAMASK,
+            transaction: {
+              ...txMeta.transaction,
+            },
+          },
+        },
+        true,
+      )
+      .catch(() => {
+        // Intentionally ignored as promise not currently used
+      });
+  }
+
+  private acceptApproval(txMeta: TransactionMeta) {
+    const id = this.getApprovalId(txMeta);
+
+    try {
+      this.messagingSystem.call('ApprovalController:acceptRequest', id);
+    } catch (error) {
+      console.error('Failed to accept transaction approval request', error);
```

Comment:
Shall we avoid panic and change both of these to `console.info` as we're trying to make this as low risk a change as possible, so if the approval requests fail, the only impact is the UI rather than any backend logic.

Reply:
I'm just thinking if a middle term isn't better like warn. Either way, move from error makes total sense.